go/rpc/encrypted_client.go

```diff
@@ -25,11 +25,6 @@ import (
 	gethlog "github.com/ethereum/go-ethereum/log"
 )
 
-const (
-	// todo: this is a convenience for testnet testing and will eventually be retrieved from the L1
-	enclavePublicKeyHex = "034d3b7e63a8bcd532ee3d1d6ecad9d67fca7821981a044551f0f0cbec74d0bc5e"
```

Comment:
This key string still appears a few times in the codebase which can probably be cleaned up? In particular can probably delete the stuff about 'EnclavePublicKey' in `transaction_injector.go`